db/schema.rb

```diff
@@ -41,6 +41,16 @@
     t.index ["course_id"], name: "index_assignments_on_course_id"
   end
 
+  create_table "codereview_matchings", force: :cascade do |t|
```

Comment:
This is definitely going to cause merge conflicts.  Can you rebase on the latest changes, and re-number whatever migrations were needed, so that there's a linearizable history for the schema?